features/steps/go_steps.rb

```diff
@@ -1,122 +1,41 @@
-require 'net/http'
+require 'os'
 
-Given("I set environment variable {string} to the app directory") do |key|
-  step("I set environment variable \"#{key}\" to \"/app/src/test/\"")
+When('I run {string}') do |scenario_name|
+  execute_command 'run-scenario', scenario_name
 end
 
-Given("I set environment variable {string} to the notify endpoint") do |key|
-  step("I set environment variable \"#{key}\" to \"http://#{current_ip}:#{MOCK_API_PORT}\"")
+When('I configure the maze endpoint') do
+  steps %(
+    When I set environment variable "DEFAULT_MAZE_ADDRESS" to "http://#{local_ip}:9339"
+  )
 end
 
-Given("I set environment variable {string} to the sessions endpoint") do |key|
```

Comment:
I'm guessing that every step definition from here downwards is removed or some way in a later change on the integration branch, as they are still used at this point of the overall change.

Could we simply reinstate them for this PR?  It would make the PR diff a lot simpler and I think help the series of changes being made hang together better.

Reply:
ok